app/src/main/res/values/strings.xml

```diff
@@ -54,6 +54,8 @@
     <string name="keep_on_title">Keep screen on while transferring</string>
     <string name="floating_text_selection_title">Floating text selection</string>
     <string name="floating_text_selection_summary">When selecting text in other apps, an option \"Send with Snapdrop\" will be shown</string>
+    <string name="show_connectivity_card_title">Display connectivity card</string>
```

Comment:
Maybe just "Connectivity banner"? 

Users might not know what a "card" UI element is, and that `enabled = show` is pretty self-explanatory IMHO

Reply:
Just pushed a small change: "Display connectivity warning" to "Connectivity warning" which is a bit shorter...